grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
+        'beginCaptures':
+          '0':
+            'name': 'string.quoted.double.js'
+          '1':
+            'name': 'punctuation.definition.string.begin.js'
+        'contentName': 'string.quoted.double.js'
+        'end': '(")|(?=\\*/)'
+        'endCaptures':
+          '0':
+            'name': 'string.quoted.double.js'
+          '1':
+            'name': 'punctuation.definition.string.end.js'
+        'name': 'source.embedded.js'
+        'patterns': [
+          {
+            'match': '\\\\(?:[^*]|\\*(?!/))'
+            'name': 'constant.character.escape.js'
+          }
+        ]
+      }
+      # Single-quoted string
+      {
+        'begin': "(')"
+        'beginCaptures':
+          '0':
+            'name': 'string.quoted.single.js'
+          '1':
+            'name': 'punctuation.definition.string.begin.js'
+        'contentName': 'string.quoted.single.js'
+        'end': "(')|(?=\\*/)"
+        'endCaptures':
+          '0':
+            'name': 'string.quoted.single.js'
+          '1':
+            'name': 'punctuation.definition.string.end.js'
+        'name': 'source.embedded.js'
+        'patterns': [
+          {
+            'match': '\\\\(?:[^*]|\\*(?!/))'
+            'name': 'constant.character.escape.js'
+          }
+        ]
+      }
+      {
+        # Nested arrays matched to keep brackets balanced
```

Comment:
Yes, because it's used to balance all that weird, noisy syntax Google use for Closure Compiler tags, which combine `< [ {` brackets with nesting and all sorts of weird-looking rubbish.